src/doc/avrdude.texi

```diff
@@ -509,8 +509,9 @@ following options are recognized:
 
 @table @code
 @item -p @var{partno}
-@cindex Option @code{-p} @var{partno}
-@cindex @code{-p} @var{partno}
+@item --part @var{partno}
+@cindex Option @code{-p ,--part} @var{partno}
```

Comment:
Should these not be two index lines?
```
@cindex Option @code{-p} @var{partname}
@cindex Option @code{--part} @var{partname}
```
(also note it's better to use partname instead of partno, as the part name is not really a number)

Multiple times. It's not so bad here, but when it's `--noerase` the reader wouldn't easily find that option in the alphabetic index as it's not sorted at n but rather at d (it trails `-D`)

Reply:
Do you instead want it to be like this?

```
@item -p @var{partname}
@item --part @var{partname}
@cindex Option @code{-p} @var{partname}
@cindex Option @code{--part} @var{partname}
@cindex @code{-p} @var{partname}
@cindex @code{--part} @var{partname}
```

Reply:
Would be brill. Many places.

Reply:
@stefanrueger done!